pom.xml

```diff
@@ -41,6 +44,7 @@
       <groupId>org.testng</groupId>
       <artifactId>testng</artifactId>
       <version>7.9.0</version>
+     <version>7.9.0</version>
```

Comment:
<div>
<h3>Code Review Agent Run <a href = https://github.com/Amruta101998/selenium-automation/pull/2#issuecomment-2123048177>#0a0bf</a> - 05/21/2024, 10:25 pm</h3>

<p>🔴 High importance</p>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Redundant version tag for 'testng' dependency is added, which could lead to Maven build warnings or errors due to multiple version definitions.
</div>
<div id="fix">
<b>Fix:</b> Remove the redundant version tag to maintain a single, clear definition of the dependency version.
</div>
<details>
<summary>
Code suggestion
</summary>
<div>
<pre><code>
 @@ -44,7 +44,6 @@
      <groupId>org.testng</groupId>
      <artifactId>testng</artifactId>
      <version>7.9.0</version>
 -    <version>7.9.0</version>
      <scope>test</scope>
    </dependency>
</code></pre>
</div>
</details>
<h4><a href="https://github.com/Amruta101998/selenium-automation/pull/2#issuecomment-2123048177"><-- Code Review Agent Run#0a0bf</a></h4>
<i>AI Code Review powered by <img src="https://bito.ai/wp-content/uploads/2023/10/Logo-Bito-Black-cropped.svg" alt="Bito Logo" width="50" height="25"></i>
</div>
<br>

<br>

</div>